package.json

```diff
@@ -7,7 +7,8 @@
     "build": "ng build",
     "watch": "ng build --watch --configuration development",
     "test": "ng test",
-    "format": "prettier \"src/**/*.{ts,html,scss,md}\" --write"
+    "format": "prettier \"src/**/*.{ts,html,scss,md}\" --write",
+    "lint": "ng lint"
```

Comment:
może warto było by dodać wykonania prettiera przed commitem? Bo niżej widzę rozjazd, np. dodanie "" oraz "," po zakończonych liniach.

Reply:
Dobry pomysł, ale wolałbym się skupić teraz na innym zadaniu, a takie usprawnienie dodać w kolejnych PRze.

Reply:
Nie bardzo rozumiem co się rozjeżdża. Pliki są przeformatowane z nowymi rule'sami.

Reply:
rzecz w tym ze skrypt format jest odpalany tylko na katalog src i temu chyba jest rozjazd... ja bym to wmergowal i zrobimy hooka na eslint i prettier.

Reply:
ok, daje approve, tylko to dodajmy :).